draft-ietf-git-using-github.md

```diff
@@ -419,6 +435,239 @@ A working group that uses GitHub MAY provide either facility at the request of
+issues.
+
+
+### Design/Editorial Labelling {#label-design}
+
+The most important distinction about an issue is whether it is substantive.  The
+labels of `design` and `editorial` are used to represent this distinction.
+
+An issue labeled as `design` has or might have a substantive effect on a
+document.  For protocol specifications, a `design` issue usually affects
+implementations or interoperability requirements.  Addressing a `design` issue
+ultimately requires Working Group consensus, even if the resolution is to make
+no change.
+
+An issue labeled as `editorial` has no substantive effect on a document, except
+to the extent that addressing the issue might make understanding the
+specification easier.  Resolution of `editorial` issues can be left to the
+discretion of editors.
+
+This distinction can be applied to all types of document.  For instance, a
+`design` issue for an Informational document might be raised to discuss
+important concepts.
+
+
+### Decision Labelling {#label-decision}
+
+Labels can be used to manage processes.  As documents mature and issues become
+more numerous, labels can be used to clearly mark the status of issues.  In
+particular, labelling of issues can be used to help in managing Working Group
+decisions.
+
+For documents that are less mature, issues with resolutions but no specific
+proposals for changes to text might be marked `editor-ready` as a way of
+signaling that there is consensus about an approach, but no specific proposal.
+Chairs might use this to signal that discussion is complete and that editors are
+to be given discretion in the construction of text.
+
+In contrast, if specific text is a prerequisite for resolving issues, as might
+be the case for more mature documents, a `proposal-ready` label might be used by
+editors to mark issues that they believe to have acceptable resolutions.
+
+For resolved issues, a `has-consensus` label might be used by chairs to mark
+issues for which formal Working Group decisions have been made (Section 6.1 of
+{{!RFC2418}}).
+
+A `v2` or `next-version` label might be used to mark and thereby save issues for
+a future version of or extension to a protocol, particularly where a resolution
+is made to take no action.
+
+
+### Component Labelling
+
+Repositories with multiple interrelated documents or a complex document with
+multiple logical components might benefit from labels that identify different
+aspects of the work.  The choice of appropriate labels for components will
+depend on the structure of specific documents.
+
+
+### Other Labels
+
+Other labels can be used depending on the needs of editors and Working Group
+processes.  For example,
+
+* An `invalid` label might be used for issues that were raised in error.
+
+* A `blocked` label might indicate an issue is awaiting resolution of an
+  external process or related issue.
+
+* A `parked` label might be used to indicate issues that do not require
+  immediate working group attention.
+
+
```

Comment:
We use a `discusss` flag for http-core to identify issues to talk about F2F.

Reply:
I considered that, but I think that it just falls into the "other other"  bucket.  We use the project boards in QUIC for doing the same thing.  I wasn't even sure about including `parked` here.